lib/rules/await-fire-event.ts

```diff
@@ -33,11 +38,28 @@ export default createTestingLibraryRule<Options, MessageIds>({
 			fireEventWrapper:
 				'Promise returned from `{{ name }}` wrapper over fire event method must be handled',
 		},
-		schema: [],
+		schema: [
+			{
+				type: 'object',
+				default: {},
+				additionalProperties: false,
+				properties: {
+					eventModule: {
+						type: 'string',
```

Comment:
Sorry, I think I didn't specify this really well in the original ticket. The idea is that this is a list accepting either "fire-event", "user-event", or both!

Frameworks like Vue Testing Library will need to report both, so they'll need to set this option to `["fire-event", "user-event"]`.

By default, it should be just `["user-event"]`

It should be fairly easy to adapt your current code tho.